### PR TITLE
[wix-style-react] - <MarketingLayout/>-  reduce spacing

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
reduce spacing

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6568

**Comment by TalZa1810** [comment]
@justinara , do we want to remove the Large example completely? 
Also, I've added a deprecation log when rendering large size. I will add also a notation within the API itself that this size is deprecated. Let me know if there are other fixes that needs to be done.

**Comment by justinara** [comment]
@TalZa1810 yes, large size will have to be removed completely

**Review by justinara — APPROVED** [review]

File: packages/wix-style-react/src/MarketingLayout/MarketingLayout.st.css [hunk 1]
```diff
@@ -9,39 +9,46 @@
...
-:global([dir="rtl"]) .root:size(large) {
-  padding-left: 30px;
-  padding-right: 60px;
+
+.root:inverted {
+  padding-inline: 30px;
+}
+
+:global([dir="rtl"]) .root {
+  direction: rtl;
```
Comment (EgoziE) [inline packages/wix-style-react/src/MarketingLayout/MarketingLayout.st.css:51]: not needed

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Pull request merged**